### DISCUSSION TITLE
BloatyNosyApp 0.60 - The Windows 11 Trash party continues nonetheless.

### DISCUSSION THREAD

**Discussion by Belim** [builtbybel/Bloatynosy#43] [Announcements]
**BloatyNosy is no longer available in the Microsoft Store and was banned by Microsoft overnight.**
Numerous update attempts have been rejected, each time with a new excuse, and now it is stated officially that the app does not have a valid certificate, even though it was inside for months before. (Previously, all Microsoft Store apps - native UWPs for example and also Win32s - were hosted and signed by the Microsoft Store and received a Microsoft signature.) The store continues to be a tragedy. It's a shame for my precious time. 
I'm annoyed. If you want me to calm down, send me a like and a dislike to @Microsoft
The download of BloatyNosy will continue to be available only here on GitHub.

**In this build, all limitations of the MS Store version have been removed once again. Additionally, the app is also provided as an MSI installer.**

<hr /><em>This discussion was created from the release <a href='https://github.com/builtbybel/BloatyNosy/releases/tag/0.60.0'>BloatyNosyApp 0.60 - The Windows 11 Trash party continues nonetheless.</a>.</em>

**Comment by CriticalO** [comment]
What bastards I only just found your app two days ago and favourited it in Git so I could install it next time I booted into Win 11 and I saw you released an update with all the features that were unavailable in the store version just 5 hours ago. And it's awesome dude I wish I'd known about this sooner. My system is nice and responsive now and all the crap has gone. The bloat ware bit says the apps are gone although if I reboot and run as administrator and do another scan the bloat ware is all back again. Edit: 0.7 version has fixed this thank you for such an awesome App.
Dude have you got a donate page preferably PayPal 😁 oh and don't let the bastards grind you down. You got Microsoft running scared lol and you called your App Nosy Bloaty. That is the funniest up yours to Microsoft's spying and bloatware I have ever seen and you even got it on the store. I know it's not there now and to be honest whoever approved it knowing what it was and what it did to win 11 has probably been sacked now anyway so hopefully that thought will cheer you up. onwards and upwards. 👍

**Reply by LyndonGingerich** [reply]
I'd rather the person who banned it be fired than the person who approved it...

**Comment by suchislife801** [comment]
Constructively speaking, I'd like to humbly recommend something they cannot block.

BloatyNosy - Powershell Edition Script.

**Comment by LyndonGingerich** [comment]
You're too good for them anyway. Keep up the good work!
